src/browser/modules/D3Visualization/components/OverviewPane.tsx

```diff
@@ -59,17 +89,39 @@ function OverviewPane({
   relationshipCount,
   stats
 }: OverviewPaneProps): JSX.Element {
+  const moreStep = 50
+  const [labelsMax, setLabelsMax] = useState(moreStep)
+  const [relationshipsMax, setRelationshipsMax] = useState(moreStep)
+
+  function onMoreClick(type: any, currentMax: number) {
+    const map: any = {
+      labels: setLabelsMax,
+      relationships: setRelationshipsMax
+    }
+    return (num: number) => map[type](currentMax + num)
+  }
+
   const { relTypes, labels } = stats
+  const visibleLabelKeys = labels ? Object.keys(labels).slice(0, labelsMax) : []
+  const visibleRelationshipKeys = relTypes
+    ? Object.keys(relTypes).slice(0, relationshipsMax)
+    : []
+  const totalNumOfLabelTypes = labels ? Object.keys(labels).length : 0
+  const totalNumOfRelTypes = relTypes ? Object.keys(relTypes).length : 0
 
   return (
     <>
-      <PaneHeader>Overview</PaneHeader>
+      <PaneHeader>{'Overview'}</PaneHeader>
```

Comment:
It's a nit but we should need the `{'`  and `'}` here right?

Reply:
Yes, it is not necessary but I added it since I like that my IDEA give it a nice color indicating that it is a text that makes it easier to spot texts in a file.